static/styles/zulip.css

```diff
@@ -42,6 +42,99 @@ a {
     cursor: pointer;
 }
 
+p.n-margin {
+    margin: 10px 0px 0px 0px;
+}
+
+.small-line-height {
+    line-height: 1.1;
+}
+
+.float-left {
+    float: left;
+}
+
+.float-right {
+    float: right;
+}
+
+.float-clear {
+    clear: both;
+}
+
+.light {
+    font-weight: 300;
+}
+
+.no-margin {
+    margin: 0;
+}
+
+.border-radius {
+    border-radius: 4px;
+}
+
+#mute_topic_container {
```

Comment:
See my earlier suggestion about renaming this to something more suggestive of notifications or oops-undo capability.